xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+   <procedure>
+    <step>
+     <para>
+      Verify that the host system is SLES 12 SP3 or newer:
+     </para>
+<screen>
+&prompt.user;cat /etc/issue
+Welcome to SUSE Linux Enterprise Server 15  (x86_64) - Kernel \r (\l).
+</screen>
+    </step>
+    <step>
+     <para>
+      Verify that the host supports <xref linkend="gloss-vt-acronym-vtd"/>
+      technology and that it is already enabled in the BIOS:
+     </para>
+<screen>
+&prompt.user;dmesg | grep -e "Directed I/O"
+[   12.819760] DMAR: Intel(R) Virtualization Technology for Directed I/O
+</screen>
+     <para>
+      If VT-d is not enabled in the BIOS, enable it and reboot the host.
+     </para>
+    </step>
+    <step>
+     <para>
+      Verify that the host has an extra GPU or VGA card:
+     </para>
+<screen>
+&prompt.user;lspci | grep -i "vga"
+07:00.0 VGA compatible controller: Matrox Electronics Systems Ltd. \
+  MGA G200e [Pilot] ServerEngines (SEP1) (rev 05)
+</screen>
+     <para>
+      With a Tesla V100 card:
+     </para>
+<screen>
+&prompt.user;lspci | grep -i nvidia
+03:00.0 3D controller: NVIDIA Corporation GV100 [Tesla V100 PCIe] (rev a1)
+</screen>
+     <para>
+      With a T1000 Mobile (available on Dell 5540):
+     </para>
+<screen>
+&prompt.user;lspci | grep -i nvidia
+01:00.0 3D controller: NVIDIA Corporation TU117GLM [Quadro T1000 Mobile] (rev a1)
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-enable-iommu">
+   <title>Enable <xref linkend="gloss-vt-acronym-iommu"/></title>
+   <para>
+    <xref linkend="gloss-vt-acronym-iommu"/> is disabled by default. You need
+    to enable it at boot time in the <filename>/etc/default/grub</filename>
+    configuration file.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      For Intel-based hosts:
+     </para>
+<screen>GRUB_CMDLINE_LINUX="intel_iommu=on iommu=pt rd.driver.pre=vfio-pci"</screen>
+     <para>
+      For AMD-based hosts:
+     </para>
+<screen>GRUB_CMDLINE_LINUX="iommu=pt amd_iommu=on rd.driver.pre=vfio-pci"</screen>
+    </step>
+    <step>
+     <para>
+      When you save the modified <filename>/etc/default/grub</filename> file,
+      re-generate the main &grub; configuration file
+      <filename>/boot/grub2/grub.cfg</filename>:
+     </para>
+<screen>&prompt.sudo;grub2-mkconfig -o /boot/grub2/grub.cfg</screen>
+    </step>
+    <step>
+     <para>
+      Reboot the host and verify that <xref linkend="gloss-vt-acronym-iommu"/>
+      is enabled:
+     </para>
+<screen>&prompt.user;dmesg |  grep -e DMAR -e IOMMU</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-blacklist-nouveau">
+   <title>Blacklist the Nouveau driver</title>
+   <para>
+    Because we want to assign the &nvidia; card to a VM guest, we need to avoid
+    use of the card by the host open source &nvidia; driver. Edit the file
+    <filename>/etc/modprobe.d/50-blacklist.conf</filename> and append the
+    following line to its end:
```

Comment:
```suggestion
    Because we want to assign the &nvidia; card to a VM guest, we need to avoid
    use of the card by the host OS's built-in driver for &nvidia; GPUs. The open
    source &nvidia; driver is called <literal>nouveau</literal>. Edit the file 
    <filename>/etc/modprobe.d/50-blacklist.conf</filename> and append the
    following line to its end:
```

Reply:
Since there are multiple nVidia drivers, the specification of "the host open source driver" seemed odd to me. Why just the open-source one? Why not disable any/all drivers? The next line makes it clear -- it's not disabling the driver completely, it's making sure that Nouveau doesn't load so that the proprietary driver **can** grab the card. As such I thought it was worth clarifying which driver we are talking about.